x-pack/plugins/security_solution/public/network/components/embeddables/translations.ts

```diff
@@ -10,7 +10,7 @@ import { i18n } from '@kbn/i18n';
 export const EMBEDDABLE_HEADER_TITLE = i18n.translate(
   'xpack.securitySolution.components.embeddables.embeddedMap.embeddableHeaderTitle',
   {
-    defaultMessage: 'Network map',
+    defaultMessage: 'Network Map',
```

Comment:
Nit to revert this, the convention is to only capitalize the first word in titles
<img width="1077" alt="Screen Shot 2022-01-19 at 9 27 21 AM" src="https://user-images.githubusercontent.com/6935300/150173243-e4b585e3-5794-4958-a3e9-563e34df2c48.png">

Reply:
fixed!